scrapy/pipelines/images.py

```diff
@@ -114,25 +114,25 @@ def file_downloaded(self, response, request, info, *, item=None):
         return self.image_downloaded(response, request, info, item=item)
 
     def image_downloaded(self, response, request, info, *, item=None):
-        
+        logger = logging.getLogger()
```

Comment:
It’s common to define this at the module level, after imports, and passing `__name__` to `getLogger` so that the logger is named after the module.